apps/server/internal/app/runtime_branch_coverage_test.go

```
@@ -0,0 +1,422 @@
+			t.Fatal(err)
+		}
+		runtimeStore.instance.Status = string(runtimepkg.StateDestroyed)
+		instance, err = service.Destroy(context.Background(), workspace.ProjectID, instance.ID)
+		if err != nil || instance.Status != string(runtimepkg.StateDestroyed) || implementation.destroyCalls != 0 {
+			t.Fatalf("Destroy() instance=%+v calls=%d err=%v", instance, implementation.destroyCalls, err)
+		}
+	})
+}
+
+func TestRuntimeInstanceServicePersistenceAndValidationGuards(t *testing.T) {
+	service, runtimeStore, implementation, workspace := runtimeServiceFixture(t)
```

golangci-lint (2.13.2)

[error] SA4006: this value of implementation is never used

(staticcheck)